### PR TITLE
doc: wifi: Update TX injection documentation for error section

### PR DESCRIPTION
**Pull request by VivekUppunda**
This commit adds error reporting section to TX injection documentation

### CODE REVIEW

File: doc/nrf/protocols/wifi/advanced_modes/raw_tx_operation.rst [hunk 1]
```diff
@@ -165,3 +165,13 @@ The channel configuration for raw packet transmission will be applied as follows
...
+
+.. _ug_nrf70_developing_error_handling_for_raw_packet_transmission:
+
+Error handling for raw packet transmission
+******************************************
+
+The raw packet transmission errors can be obtained by invoking the ``NET_REQUEST_STATS_GET_WIFI`` network management API.
+The ``error`` member in ``struct net_stats_wifi`` will provide the transmit errors for raw packet transmit failure.
+
+you can refer the ``struct net_stats_wifi`` in :file:`zephyr/include/zephyr/net/net_stats.h`
```
Comment (kittydepa) [inline doc/nrf/protocols/wifi/advanced_modes/raw_tx_operation.rst:177]:
```suggestion
You can refer the ``struct net_stats_wifi`` struct  in the :file:`zephyr/include/zephyr/net/net_stats.h` header file.
```

File: doc/nrf/protocols/wifi/advanced_modes/raw_tx_operation.rst [hunk 2]
```diff
@@ -165,3 +165,13 @@ The channel configuration for raw packet transmission will be applied as follows
...
+
+.. _ug_nrf70_developing_error_handling_for_raw_packet_transmission:
+
+Error handling for raw packet transmission
+******************************************
+
+The raw packet transmission errors can be obtained by invoking the ``NET_REQUEST_STATS_GET_WIFI`` network management API.
+
+After invoking the API, you can use the ``struct net_stats_wifi`` struct in the :file:`zephyr/include/zephyr/net/net_stats.h` header file.
+The ``error`` member in ``struct net_stats_wifi`` will provide the transmit errors for raw packet transmit failure.
```
Comment (krish2718) [inline doc/nrf/protocols/wifi/advanced_modes/raw_tx_operation.rst:177]:
```suggestion
The ``error`` member in ``struct net_stats_wifi`` will provide the transmit errors for all packets including raw packet transmit failure.
```

**Review by krish2718 — APPROVED** [review]

**Comment by richabp** [comment]
Discussed offline, that the struct need not to link to Zephyr: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/connectivity/networking/api/net_stats.html#c.net_stats_wifi. 
@VivekUppunda : Please confirm.

**Review by richabp — APPROVED** [review]

**Comment by VivekUppunda** [comment]
> Discussed offline, that the struct need not to link to Zephyr: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/connectivity/networking/api/net_stats.html#c.net_stats_wifi. @VivekUppunda : Please confirm.

Hi @richabp - since the doxygen description in the page would probably need updating, it is better to guide the reader of the document to the actual structure.
Thanks and regards
Vivek

**Pull request merged**